service/worker/scheduler/workflow_test.go

```diff
@@ -1491,6 +1492,7 @@ func (s *workflowSuite) TestExitScheduleWorkflowWhenNoActions() {
 	s.expectStart(func(req *schedspb.StartWorkflowRequest) (*schedspb.StartWorkflowResponse, error) {
 		s.True(time.Date(2022, 6, 1, 0, 30, 0, 0, time.UTC).Equal(s.now()))
 		s.Equal("myid-2022-06-01T00:30:00Z", req.Request.WorkflowId)
+		lastScheduleTime = time.Now()
```

Comment:
why can't you change expectStart, etc. to set ActualTime from the workflow time?

Reply:
just did that :-) misunderstood your comment earlier